addons/im_livechat/report/im_livechat_report_channel.py

```diff
@@ -239,6 +239,13 @@ def formatted_read_group(self, domain, groupby=(), aggregates=(), having=(), off
             )
         return result
 
+    def _read_group_orderby(self, order: str, groupby_terms: dict[str, SQL], query) -> SQL:
```

Comment:
If we really want to type it, then we should do it for every parameter (including query).